js_modules/dagit/packages/core/src/launchpad/LaunchpadSingleSessionContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+import {
+  createSingleSession,
+  IExecutionSessionChanges,
+  useInitialDataForMode,
+} from '../app/ExecutionSessionStorage';
```

Comment:
I wonder if this file should also move to the /launchpad directory

Reply:
Sure, will do this in a followup to limit the blast radius of this PR.